js/components/skjema/periodevelger/PeriodeDatoinput.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import cn from 'classnames';
+import PropTypes from 'prop-types';
+import MaskedInput from 'react-maskedinput';
+import { fieldPropTypes } from '../../../propTypes';
+
+const PeriodeDatoinput = ({ meta, input, id, onDoubleClick, withRef }) => {
+    const classNames = cn('skjemaelement__input datovelger__input', {
+        'datovelger__input--fom': input.name.indexOf('fom') > -1,
+        'datovelger__input--tom': input.name.indexOf('tom') > -1,
+        'skjemaelement__input--harFeil': meta.touched && meta.error,
+    });
+
+    return (<MaskedInput
+        ref={withRef}
+        type="tel"
```

Comment:
skal det være `tel`?

Reply:
Tror det er en hack for å få en spessiel oppførsel. Husker ikke helt hva som var greia :P

Reply:
Ja, det gir numerisk tastatur på mobil.